q2_demux/plugin_setup.py

```diff
@@ -135,7 +138,9 @@
               'average positional qualities across all of the sequences '
               'selected. If input sequences are paired end, plots will be '
               'generated for both forward and reverse reads for the same `n` '
-              'sequences.')
+              'sequences.'),
+        'delta': ('The acceptable delta cutoff for disparities in sequence '
+                  'length')
```

Comment:
I was on the same page as @ebolyen up until now, but @jairideout makes a good point about assuming the user's understanding of the situation. I am fine with dropping the parameter and sticking with displaying the warning whenever it is truncated. Maybe we can massage the alert a bit to prevent the wolf-fatigue that @ebolyen points out (which is still a concern to me).